omnigan/generator.py

```diff
@@ -82,7 +97,7 @@ def __init__(self, opts, latent_shape=None, verbose=None):
             self.decoders["s"] = SegmentationDecoder(opts)
 
         if "m" in opts.tasks and not opts.gen.m.ignore:
-            self.decoders["m"] = MaskDecoder(opts)
+            self.decoders["m"] = MaskDecoder(opts)  # DeeplabDecoder()
```

Comment:
what is deeplab decoder if not the segmentation decoder?

Reply:
I ended up keeping our baseDecoder as the decoder even if Deeplab encoder was chosen. 
Do you mean you'd like to have some decoder with atrous convolutions ?

Reply:
I mean the "SegmentationDecoder" a couple lines above should be your "DeeplabDecoder" no? the latter is not a candidate for the masker